.github/workflows/ci_vlmd.yaml

```diff
@@ -2,13 +2,12 @@ name: CI Workflow
 
 on:
   push:
-    branches:
-      - master
     paths:
-      - vlmd-submission-tools/
+      - vlmd-submission-tools/**
+      - .github/workflows/ci_vlmd.yaml
   pull_request:
-    branches:
-      - master
+    paths:
+      - vlmd-submission-tools/
 
 jobs:
   test:
```

Comment:
nitpicking: should consider change this job's name to something like `vlmd-wf-test` so easier to infer from the status page what this action iis testing for

Reply:
Good idea!

Reply:
hmmm does not see the job get triggered after new commit 🤔

Reply:
Oh, interesting. I think the test got triggered for the [commit for the name-change](https://github.com/uc-cdis/containers/actions/runs/6473862155). The next commit was for a conflict in the root-level secrets.baseline file. Other tests were triggered on that commit. I did another commit in the vlmd-submission-tools. It [triggered the ci_vlmd](https://github.com/uc-cdis/containers/actions/runs/6474317608).

Reply:
that is right! thanks